src/Component/Histories/container.js

```diff
@@ -16,11 +22,15 @@ const mapStateToProps = state => ({
 const mapDispatchToProps = dispatch => ({
   getHistories: compose(dispatch, getHistories),
   getNextHistories: compose(dispatch, getNextHistories),
+  clear: compose(dispatch, clear),
 })
 
-const didMount = ({ getHistories }) => getHistories()
-
-const lifecycles = componentDidMount(didMount)(Histories)
+const lifecycles = compose(
+  componentDidMount(() => window.document.querySelector('body').classList.add(pageName)),
+  componentDidMount(({ getHistories }) => getHistories()),
+  componentWillUnmount(() => window.document.querySelector('body').classList.remove(pageName)),
+  componentWillUnmount(({ clear }) => clear()),
+)(Histories)
```

Comment:
Why not running the classList changes in a dedicated epic attached to getHistories/clear actions?

Reply:
Also, better do it in a single event as, everytime you add a componentDidMount/etc... the result is that you'll create a wrapper class component around your input component with the event listener attached.
The result would be:
```javascript
<ComponentWillUnMount>
  <ComponentWillUnMount>
    <ComponentDidMount>
      <ComponentDidMount>
        <Histories />
      </ComponentDidMount>
    </ComponentDidMount>
  </ComponentWillUnMount>
</ComponentWillUnMount>
```